index.bs

```diff
@@ -862,19 +906,9 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Increment |sourceToAttribute|'s [=attribution source/number of reports=] value by 1.
 1. If |trigger|'s [=attribution trigger/dedup key=] is not null, [=list/append=] it to |sourceToAttribute|'s
     [=attribution source/dedup keys=].
-1. Let |rateLimitRecord| be a new [=attribution rate-limit record=] with the items:
-
-    : [=attribution rate-limit record/source site=]
-    :: |sourceToAttribute|'s [=attribution source/source site=]
-    : [=attribution rate-limit record/attribution destination=]
-    :: |attributionDestination|
-    : [=attribution rate-limit record/reporting endpoint=]
-    :: |sourceToAttribute|'s [=attribution source/reporting endpoint=]
-    : [=attribution rate-limit record/trigger time=]
-    :: |trigger|'s [=attribution trigger/trigger time=]
 1. Add |rateLimitRecord| to the [=attribution rate-limit cache=].
```

Comment:
Do we want to move the "add to cache" step to a separate algorithm so we don't duplicate clearing old entries line?

Reply:
We can reorganize that in a followup.